onrobot_rg2_driver/src/rg2_driver_node.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python3
+from modbus_tk.modbus import ModbusError
+from threading import Lock
+
+import rospy
+
+from sensor_msgs.msg import JointState
+
+from onrobot_rg2_driver.srv import GripperState, GripperStateResponse, GripperStateRequest
+
+from rg2_modbus import Rg2Modbus
+
+
+class Rg2DriverNode:
+    MIN_RG2_FORCE: float = 3.0  # [N]
+    MAX_RG2_FORCE: float = 40.0  # [N]
+    MIN_RG2_RANGE: float = 0.0  # [N]
+    MAX_RG2_RANGE: float = 100.0  # [N]
+    TIMEOUT: float = 10.0  # [s]
+
+    def __init__(self) -> None:
+        rospy.init_node("rg2_driver_node", anonymous=False)
+
+        ur_robot_ip = rospy.get_param('~ur_robot_ip', '10.42.0.49')
```

Comment:
```suggestion
        ur_robot_ip = rospy.get_param('~ur_robot_ip', '10.15.20.4')
```
according to readme this is deafult